app/scripts/views/sign_up.js

```diff
@@ -177,6 +177,14 @@ define(function (require, exports, module) {
         shouldFocusPassword: autofocusEl === 'password',
         showSyncSuggestion: this.isSyncSuggestionEnabled()
       };
+      if (this.getUserAgent()._hasWebChannelSupport()) {
+        context.escapedSyncSuggestionUrl = encodeURI('https://accounts.firefox.com/signup?service=sync&context=fx_desktop_v3&entrypoint=uitour');
```

Comment:
I believe that since this is basically pointing to the same server, just a different route, the scheme and authority could be removed, so it would just be `encodeURI('/signup?service=sync&context=fx_desktop_v3&entrypoint=uitour');`. @shane-tomlinson is that correct?